api/api/validator.py

```diff
@@ -18,6 +18,7 @@
 _base64 = re.compile(r'^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$')
 _boolean = re.compile(r'^true$|^false$')
 _dates = re.compile(r'^\d{8}$')
+_daemon_names = re.compile(r'^wazuh-analysisd$|^wazuh-remoted$|^wazuh-db$')
```

Comment:
Why are we using regex here? It could be an enum in the spec, increasing maintainability and the output since it is always better to understand that a word does not belong to a list than to a regex.

Reply:
Changed to SPEC enum